Jenkinsfile

```diff
@@ -750,6 +750,101 @@ pipeline {
+            model.decoder.masked_softmax_fusion=False \
+            model.decoder.bias_activation_fusion=False \
+            model.decoder.activations_checkpoint_method='block' \
+            model.decoder.activations_checkpoint_num_layers=1 \
+            model.micro_batch_size=2 \
+            model.global_batch_size=4 \
+            model.train_ds.src_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-de-en.src \
+            model.train_ds.tgt_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-de-en.ref \
+            model.validation_ds.src_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-de-en.src \
+            model.validation_ds.tgt_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-de-en.ref \
+            ~model.test_ds \
+            model.train_ds.dataset_type=text_memmap \
+            model.encoder_tokenizer.library=sentencepiece \
+            model.encoder_tokenizer.model=/home/TestData/nlp/nmt/toy_data/spm_64k_all_langs_plus_en.model \
+            model.decoder_tokenizer.library=sentencepiece \
+            model.decoder_tokenizer.model=/home/TestData/nlp/nmt/toy_data/spm_64k_all_langs_plus_en.model \
+            +model.hidden_size=64"
+            sh "python examples/nlp/machine_translation/adapter_tuning/nmt_adapter_tuning.py \
+            trainer.devices=2 \
+            trainer.accelerator=gpu \
+            trainer.log_every_n_steps=1 \
+            trainer.val_check_interval=10 \
+            +trainer.limit_val_batches=2 \
+            trainer.accumulate_grad_batches=1 \
+            trainer.max_steps=10 \
+            trainer.precision=16 \
+            trainer.gradient_clip_val=1.0 \
+            exp_manager.exp_dir=examples/nlp/machine_translation/megatron_nmt_results_adapters \
+            model.tensor_model_parallel_size=1 \
+            model.seq_length=128 \
+            model.micro_batch_size=2 \
+            model.global_batch_size=4 \
+            model.train_ds.src_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-de-en.src \
+            model.train_ds.tgt_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-de-en.ref \
+            model.validation_ds.src_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-de-en.src \
+            model.validation_ds.tgt_file_name=/home/TestData/nlp/nmt/toy_data/wmt14-de-en.ref \
+            ~model.test_ds \
+            model.train_ds.dataset_type=text_memmap \
+            model.encoder_tokenizer.library=sentencepiece \
+            model.encoder_tokenizer.model=/home/TestData/nlp/nmt/toy_data/spm_64k_all_langs_plus_en.model \
+            model.decoder_tokenizer.library=sentencepiece \
+            model.decoder_tokenizer.model=/home/TestData/nlp/nmt/toy_data/spm_64k_all_langs_plus_en.model \
+            model.adapter_tuning.adapter_dim=4 \
+            model.adapter_tuning.adapter_dropout=0.1 \
+            model.pretrained_model_path=examples/nlp/machine_translation/megatron_nmt_results/megatron_nmt/checkpoints/megatron_nmt.nemo \
+            +model.hidden_size=64"
```

Comment:
Why do you need `+model.hidden_size` here? I would recommend not adding this and keep args specific to encoder and decoder.